packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -2474,6 +2474,10 @@ export class LitNodeClientNodeJs
 
     // Compute the address from the public key if it's provided. Otherwise, the node will compute it.
     const pkpEthAddress = (function () {
+
+      // prefix '0x' if it's not already prefixed
+      params.pkpPublicKey = hexPrefixed(params.pkpPublicKey!);
```

Comment:
Good catch, however I would prefer we use another value if possible so we don't modify the caller object property

Reply:
Makes sense!